appcues/src/main/java/com/appcues/trait/appcues/TooltipTrait.kt

```diff
@@ -260,8 +294,16 @@ internal class TooltipTrait(
         )
     }
 
-    private fun Modifier.tooltipSize(style: ComponentStyle?): Modifier = then(
-        Modifier.size(width = style?.width?.dp ?: Dp.Unspecified, height = style?.height?.dp ?: Dp.Unspecified)
+    private fun Modifier.tooltipSize(style: ComponentStyle?, maxWidth: Dp, maxHeight: Dp): Modifier = then(
+        if (style?.width == null) Modifier.requiredWidthIn(
+            min = 48.dp,
+            max = min(400.dp, maxWidth)
+        ) else Modifier.width(width = style.width.dp)
+    ).then(
+        if (style?.height == null) Modifier.requiredHeightIn(
+            min = 48.dp,
+            max = max(48.dp, maxHeight)
+        ) else Modifier.height(height = style.height.dp)
```

Comment:
oh ok I see the defaults are applied here

Reply:
yeah I took out the constants since I'm not using anywhere else, also trying to follow some rules per discussed [here](https://appcues.slack.com/archives/C031RLGS4F8/p1677594509754279?thread_ts=1677533504.378769&cid=C031RLGS4F8)